app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -227,6 +227,21 @@ struct DeviceDelegateOpenXR::State {
     viewConfig.resize(viewCount, {XR_TYPE_VIEW_CONFIGURATION_VIEW});
     CHECK_XRCMD(xrEnumerateViewConfigurationViews(instance, system, viewConfigType, viewCount, &viewCount, viewConfig.data()));
 
+#ifdef PICOOXR
+    // The Pico 4 is much more capable than it advertises via OpenXR, and the primary device used
+    // with Pico OXR builds. We thus bump resulutions by 1.4 for a drastic improvement in image clarity.
+    if (viewCount > 0) {
+      viewConfig.front().recommendedImageRectWidth *= 1.4f;
+      viewConfig.front().recommendedImageRectHeight *= 1.4f;
+      if (viewConfig.front().recommendedImageRectWidth > viewConfig.front().maxImageRectWidth) {
+        viewConfig.front().recommendedImageRectWidth = viewConfig.front().maxImageRectWidth;
```

Comment:
You could do 
`viewConfig.front().recommendedImageRectWidth = std::min(viewConfig.front().maxImageRectWidth, viewConfig.front().recommendedImageRectWidth);`

Same for the max size